### ISSUE TITLE
Cannot set ymin ymax

### ISSUE THREAD

**Issue by joedborg** [loony-bean/textplots-rs#7]
I cannot set these and the defaults aren't always pretty:

```
⡁⣾    ⣾    ⡀          ⣾  ⢀⢾       ⢰⠉⡇     204.3
⠄⣿   ⡸⢸   ⢸⡇         ⢠⢻  ⡎⢸       ⢸ ⡇    
⠂⣿   ⡇⢸   ⢸⡇  ⢀      ⡸⢸  ⡇⢸       ⢸ ⡇    
⡁⣿   ⡇⢸   ⢸⡇  ⣿      ⡇⢸  ⡇⢸       ⢸ ⡇    
⠄⣿   ⡇⢸   ⢸⡇  ⣿      ⡇⢸  ⡇⢸       ⢸ ⡇    
⠂⠏⡆  ⡇⢸   ⢸⢣  ⣿      ⡇⢸  ⡇⢸  ⡀    ⢸ ⢱    
⡁ ⡇  ⡇⢸   ⢸⢸ ⢀⢿      ⡇⢸  ⡇⢸ ⢸⡇    ⢸ ⢸    
⠄ ⡇  ⡇⢸   ⢸⢸ ⢸⢸      ⡇⢸  ⡇⢸ ⢸⡇    ⢸ ⢸    
⠂ ⡇  ⡇⢸   ⢸⢸ ⢸⢸      ⡇⢸  ⡇⢸ ⡇⡇    ⡎ ⢸    
⡁ ⡇  ⡇⢸ ⢀⡄⢸⢸ ⢸⢸      ⡇⢸  ⡇⢸ ⡇⡇   ⢰⠁ ⢸    
⠄ ⠑⠒⠒⠃⠈⠒⠃⠑⠚⠈⠒⠚⠘⠤⠒⠒⠒⠒⠒⠃⠈⠒⠒⠃ ⠉⠁⠑⠒⠒⠒⠚  ⠈⠒⠒⠒⠄
⠂                                        
⡁                                        
⠄                                        
⠂                                        
⡁                                        
⠄                                        
⠂                                        
⡁                                        
⠄                                        
⠁⠈ ⠁⠈ ⠁⠈ ⠁⠈ ⠁⠈ ⠁⠈ ⠁⠈ ⠁⠈ ⠁⠈ ⠁⠈ ⠁⠈ ⠁⠈ ⠁⠈ ⠁  10.0
```
I would like to be able to set these as well as perhaps a method to shrink wrap to the min and max from the slice.

**Comment by loony-bean** [comment]
Thanks for reporting! I've fixed the auto scaling issue in 0.4.1
Do you think there is still a need to manually set the y range?

**Comment by loony-bean** [comment]
Yep, that's possible
